tests/trace/test_exporter.py

```diff
@@ -5,7 +5,7 @@
 def test_exporter(monkeypatch: pytest.MonkeyPatch):
     # Test that it defaults to local
     exporter = HttpExporter()
-    assert exporter._base_url == "http://0.0.0.0:6006"
+    assert exporter._base_url == "http://127.0.0.0:6006"
```

Comment:
```suggestion
    assert exporter._base_url == "http://127.0.0.1:6006"
```

Reply:
Good catch

Reply:
Good catch, addressed in [bab000c](https://github.com/Arize-ai/phoenix/pull/2161/commits/bab000cc17ff13c7228a381606021e49c70c0b9b)